test/google_search_data_viewer/keywords/keyword_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule GoogleSearchDataViewer.Keywords.KeywordTest do
+  use GoogleSearchDataViewer.DataCase, async: true
+
+  alias GoogleSearchDataViewer.Keywords.Keyword
+
+  describe "create_keyword_uploads/2" do
+    test "give a valid list of keywords and a user, creates keywords for the user." do
+      user = insert(:user)
+      keywords = ["dog", "cat", "fish"]
+
+      assert Keyword.create_keyword_uploads(keywords, user) == length(keywords)
```

Comment:
```suggestion
      assert Keyword.create_keyword_uploads(keywords, user) == Enum.count(keywords)
```

We prefer to use `Enum.count/1` instead of length.